src/api/db/schemas/Image.js

```diff
@@ -13,6 +14,13 @@ const Schema = mongoose.Schema;
  *    userSetData  - user configured EXIF data
  */
 
+const ImageCommentSchema = new Schema({
+  _id: { type: String, required: true, default: randomUUID },
```

Comment:
Is there a reason to manually set the `_id` here? Unless we want the _ids to be something other than one of Mongoose's randomly generated `ObjectIds`, I think if we leave the _id field out entirely Mongoose will [add it for us](https://mongoosejs.com/docs/guide.html#_id).